cmd/repl/repl.go

```diff
@@ -40,11 +40,22 @@ import (
 
 const tablePageSize = 1000
 
+func lineIsComment(line string) bool {
+	line = strings.TrimSpace(line)
+	if line == "--" || strings.HasPrefix(line, "-- ") {
```

Comment:
Does SQL syntax state that `--` plus whitespace indicate the beginning of a comment?

Reply:
It's a MySQL extension. Because it also works on other SQL dialects, it may be reasonable for SQLFlow to accept this extension, thus making SQLFlow programs with comments portable to all other SQL dialects.